xbmc/addons/AddonStatusHandler.h

```diff
@@ -37,6 +41,7 @@ namespace ADDON
 
     private:
       static CCriticalSection   m_critSection;
+      const uint32_t m_instanceID;
```

Comment:
initilialize the member here

Reply:
Is in constructor, without there the `const` brings compile error.